streetwhisperapp.py

```diff
@@ -118,7 +167,30 @@ def questions_ui(diarize_model):
     # Input file
     rprint("[blue]=============================[blue]")
     rprint(f"[bold]Enter the absolute path to the audio file you want to do the \"{process_selected['process_selected']}\" process on:[bold]")
-    input_file = input()
+
+    # Check if audio file path is a valid path within system that CLI is running from
+
+    while True:
+        input_file = input()
+        input_audio_path = input_file.strip()  # remove leading and trailing whitespace
+        # Check 1 for input file: Validate whether path is valid
+        audio_path_last_backslash_index = input_file.rfind("/")
+        audio_name = input_file[audio_path_last_backslash_index + 1:]
+        audio_name = audio_name.strip()  # remove leading and trailing whitespace
```

Comment:
If you have a path like: "/Users/name/downloads/    audio file with space at beginning" where there is whitespace at the beginning of the audio file, the system won't be able to find the file if you strip the leading whitespace. I think you can skip the step of striping whitespace from the audio name itself and coming it to make the new input_audio_path and just strip the whitespace from the input audio PATH only.

Reply:
@moonsdust Great catch, thanks a lot for your help!